addons/binding/org.openhab.binding.loxone/src/main/java/org/openhab/binding/loxone/handler/LoxoneMiniserverHandler.java

```diff
@@ -0,0 +1,574 @@
+                    if ((OnOffType) command == OnOffType.ON) {
+                        if (control instanceof LxControlPushbutton) {
+                            ((LxControlPushbutton) control).pulse();
+                        } else {
+                            ((LxControlSwitch) control).on();
+                        }
+                    } else {
+                        ((LxControlSwitch) control).off();
+                    }
+                }
+                return;
+            }
+
+            if (control instanceof LxControlJalousie) {
+                LxControlJalousie jalousie = (LxControlJalousie) control;
+                if (command instanceof PercentType) {
+                    jalousie.moveToPosition(((PercentType) command).doubleValue() / 100);
+                } else if (command instanceof UpDownType) {
+                    if ((UpDownType) command == UpDownType.UP) {
+                        jalousie.fullUp();
+                    } else {
+                        jalousie.fullDown();
+                    }
+                } else if (command instanceof StopMoveType) {
+                    if ((StopMoveType) command == StopMoveType.STOP) {
+                        jalousie.stop();
+                    }
+                }
+                return;
+            }
+
+            if (control instanceof LxControlLightController) {
+                LxControlLightController controller = (LxControlLightController) control;
+                if (command instanceof OnOffType) {
+                    if ((OnOffType) command == OnOffType.ON) {
+                        controller.allOn();
+                    } else {
+                        controller.allOff();
+                    }
+                } else if (command instanceof UpDownType) {
+                    if ((UpDownType) command == UpDownType.UP) {
+                        controller.nextScene();
+                    } else {
+                        controller.previousScene();
+                    }
+                } else if (command instanceof DecimalType) {
+                    controller.setScene(((DecimalType) command).intValue());
+                }
+                return;
+            }
+
+            if (control instanceof LxControlRadio) {
+                LxControlRadio radio = (LxControlRadio) control;
+                if (command instanceof OnOffType) {
+                    if ((OnOffType) command == OnOffType.OFF) {
+                        radio.setOutput(0);
+                    }
+                } else if (command instanceof DecimalType) {
+                    radio.setOutput(((DecimalType) command).intValue());
+                }
+                return;
+            }
+
+            logger.debug("Incompatible operation on control {}", control.getUuid().toString());
+
+        } catch (IOException e) {
+            updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.COMMUNICATION_ERROR, e.getMessage());
+        } catch (Exception e) {
```

Comment:
Do you really need to catch the Exception-class or are there more specific exceptions you could catch like the IOException or maybe only RuntimeException's (which is not that good either). The current code and also catching RuntimeException's might 'hide' programming errors like NullPointerExceptions occurring.

https://www.google.nl/search?q=java%20why%20not%20to%20catch%20exception